### ISSUE TITLE
Can not open UE5 created projects with Rider (IDE)

### ISSUE THREAD

**Issue by TomCoursow** [AchetaGames/Epic-Asset-Manager#195]
**Describe the bug**
If I used the downloaded (via EpicAssetManager) UE5 Version to create a C++ Project. The projects builds / compiles fine in engine.
But if I try to open the project with Jetbrains Rider (IDE) the project does not load at all.

**To Reproduce**
1. Have Arch Based Linux Distribution
2. Install dependencies: mono, mono-msbuild, dotnet-sdk
3. Install Unreal Engine 5.0.2
4. Start up the Engine
5. Create a new C++ Project
6. Project is building / compiling successfully using the engine itself...
7. Go to Editor -> Editor Preferences -> Source Code
8. Set Rider as Source Code Acessor
9. In Unreal Engine: Tools -> Refresh Rider Uproject Files
10. Load the Project with Rider by opening the .uproject File

**Expected behavior**
The Newly created UE5 C++ Project is successfully loaded

** What happens instead**
The project can not be loaded at all.

**Desktop:**
 - OS: EndeavourOS
 - System Libs: mono, mono-msbuild, dotnet-sdk
 - The UE5 Bundles mono, dotnet and msbuild libs seems to be there as well (hooked these up in Rider)
 - Browser: Google-Chrome
 - Epic Asset Manager Version: 3.5.0 (Flatpak)

**Unreal Build Tool Tab (in Rider):**
```
Setting up Unreal Engine 5 project files...

Setting up bundled DotNet SDK
/home/tom_coursow/UnrealEngine/dev-5.0.2/Engine/Binaries/ThirdParty/DotNet/Linux/sdk/3.1.401/Microsoft.Common.CurrentVersion.targets(4167,5): warning MSB3245: Could not resolve this reference. Could not locate the assembly "Ionic.Zip.Reduced". Check to make sure the assembly exists on disk. If this reference is required by your code, you may get compilation errors. [/home/tom_coursow/UnrealEngine/dev-5.0.2/Engine/Source/Programs/UnrealBuildTool/UnrealBuildTool.csproj]
Platform/Android/AndroidAARHandler.cs(13,7): error CS0246: The type or namespace name 'Ionic' could not be found (are you missing a using directive or an assembly reference?) [/home/tom_coursow/UnrealEngine/dev-5.0.2/Engine/Source/Programs/UnrealBuildTool/UnrealBuildTool.csproj]
Platform/IOS/IOSToolChain.cs(14,7): error CS0246: The type or namespace name 'Ionic' could not be found (are you missing a using directive or an assembly reference?) [/home/tom_coursow/UnrealEngine/dev-5.0.2/Engine/Source/Programs/UnrealBuildTool/UnrealBuildTool.csproj]
Platform/IOS/IOSToolChain.cs(15,7): error CS0246: The type or namespace name 'Ionic' could not be found (are you missing a using directive or an assembly reference?) [/home/tom_coursow/UnrealEngine/dev-5.0.2/Engine/Source/Programs/UnrealBuildTool/UnrealBuildTool.csproj]
Platform/Mac/MacToolChain.cs(11,7): error CS0246: The type or namespace name 'Ionic' could not be found (are you missing a using directive or an assembly reference?) [/home/tom_coursow/UnrealEngine/dev-5.0.2/Engine/Source/Programs/UnrealBuildTool/UnrealBuildTool.csproj]
Platform/Mac/UEDeployMac.cs(10,7): error CS0246: The type or namespace name 'Ionic' could not be found (are you missing a using directive or an assembly reference?) [/home/tom_coursow/UnrealEngine/dev-5.0.2/Engine/Source/Programs/UnrealBuildTool/UnrealBuildTool.csproj]
Platform/TVOS/TVOSToolChain.cs(13,7): error CS0246: The type or namespace name 'Ionic' could not be found (are you missing a using directive or an assembly reference?) [/home/tom_coursow/UnrealEngine/dev-5.0.2/Engine/Source/Programs/UnrealBuildTool/UnrealBuildTool.csproj]
Platform/TVOS/TVOSToolChain.cs(14,7): error CS0246: The type or namespace name 'Ionic' could not be found (are you missing a using directive or an assembly reference?) [/home/tom_coursow/UnrealEngine/dev-5.0.2/Engine/Source/Programs/UnrealBuildTool/UnrealBuildTool.csproj]
GenerateProjectFiles ERROR: Failed to build UnrealBuildTool
```

**Problems Tab (in Rider):**
```
Solution 'Rider1' load failed
```
```
Failed to generate project model definition files for the project file.
```

**Rider Toolset and Build Settings:**
![image](https://user-images.githubusercontent.com/17900090/173347240-1b1c860e-6937-4455-91ce-13d75535bdb0.png)

Configured all Unreal Bundled mono, dotnet and msbuild paths...
But I also tried switching all 3 main path to my system related mono / mono-sgen, dotnet and mono-msbuild & dotnet msbuild paths...
**All seem to have the same issue with the Ionic DLL...**
The **Ionic.Zip.Reduced.dll** itself seems to be present in 2 of the engine directories...

* dev-5.0.2/Engine/Binaries/DotNET/AutomationTool/Ionic.Zip.Reduced.dll
* dev-5.0.2/Engine/Binaries/DotNET/UnrealBuildTool/Ionic.Zip.Reduced.dll

**Jetbrains Rider Support Contact:**
The Jetbrains support told me that this is probably not a Rider issue but some UE5 Dependency issue...
( https://forums.unrealengine.com/t/unresolved-references/314030 )
I found a Setup.sh in **<EnginesHome>/dev-5.0.2/Engine/Build/BatchFiles/Linux** but if I execute that file I get:
```
Installing a bundled clang toolchain
./Setup.sh: Line 47: ./SetupToolchain.sh: Not found File or Directory
```
**Additional Info:**
UE4.27.2 Engine (downloaded via EpicAssetManager) created projects work fine...

**Comment by TomCoursow** [comment]
I would liek to prefer to not build UE5 from source with that huge Repo...
Is my issue somehow related to the Binary Install and can this be fixed or is this a general UE5 issue?

**Comment by aknarts** [comment]
The fix will come in 5.0.3 so as soon as it gets released this should get resolved.
